runtime/src/main/java/io/micronaut/reactive/reactor/instrument/ReactorInstrumentation.java

```diff
@@ -47,14 +48,13 @@
     /**
      * Initialize instrumentation for reactor with the tracer and factory.
      *
-     * @param beanContext   The bean context
      * @param instrumenterFactory The instrumenter factory
      */
     @SuppressWarnings("unchecked")
     @PostConstruct
-    void init(BeanContext beanContext, ReactorInstrumenterFactory instrumenterFactory) {
-        Schedulers.onScheduleHook("reactor-micronaut-instrumentation", runnable -> {
-            if (instrumenterFactory.hasInstrumenters()) {
+    void init(ReactorInstrumenterFactory instrumenterFactory) {
+        if (instrumenterFactory.hasInstrumenters()) {
+            Schedulers.onScheduleHook("reactor-micronaut-instrumentation", runnable -> {
```

Comment:
Shouldn't this be removed in `@PreDestroy`?

Reply:
Added to the previous PR